libavfilter/vf_dnn_processing.c

```diff
@@ -45,7 +45,7 @@ typedef struct DnnProcessingContext {
 #define OFFSET(x) offsetof(DnnProcessingContext, dnnctx.x)
 #define FLAGS AV_OPT_FLAG_FILTERING_PARAM | AV_OPT_FLAG_VIDEO_PARAM
 static const AVOption dnn_processing_options[] = {
-    { "dnn_backend", "DNN backend",                OFFSET(backend_type),     AV_OPT_TYPE_INT,       { .i64 = 0 },    INT_MIN, INT_MAX, FLAGS, "backend" },
+    { "dnn_backend", "DNN backend",                OFFSET(backend_type),     AV_OPT_TYPE_INT,       { .i64 = 2 },    INT_MIN, INT_MAX, FLAGS, "backend" },
```

Comment:
what will happen when we don't build with tf/vino.

Reply:
There would be no possible values for backend_type like 'tensorflow', 'openvino', so it would report error like `failed to parse option backed_type with tensorflow/openvino`.

Reply:
Hi Yejun, I just updated commits. Does this error info make sense to you?
If so, I would like to update it in community.